docker-compose.services.yml

```diff
@@ -1,6 +1,10 @@
 version: '3.9'
-services:
 
+networks:
```

Comment:
if you move networks there, may be it is a good idea to move volumes there as well?

Reply:
no it's a common practice to leave volumes beneath the file